app/render/index.ts

```diff
@@ -255,45 +271,69 @@ const nightPDF = (async function () {
 		page: number | null = null,
 	) => {
 		console.log("opening ", file);
-		if (_pdfElement.src) {
-			console.log("opening in new window");
-			window.api.newWindow(file);
+		const tabs = _tabGroup?.getTabs();
+		if (!tabs || tabs.length === 0) {
+			console.debug("No tabs yet, creating new tab");
+		}
+		let resolved_file;
+		let title: string;
+		console.debug(file);
+		if (typeof file === "string") {
+			resolved_file = await window.api.ResolvePath(file);
+			title = file;
```

Comment:
```suggestion
			title = await window.api.getPath(file[0]);
```

getPath(will probably rename it after merging this to getFileName) returns the name of the file, I feel this is better to returning just the absolute path which can make things look rather ugly

Reply:
Yeah, I'll take a look this evening, just off the top of my head (I'm not at my PC) having just the filename is probably the nicest, but the absolute (resolved, in case of symlinks) should be stored somewhere to prevent the same file opening Multiple times (I could add it to the map for css injection and rename it to something more generic like "documentProperties" or "tabProps..."

Reply:
> Yeah, I'll take a look this evening, just off the top of my head (I'm not at my PC) having just the filename is probably the nicest, but the absolute (resolved, in case of symlinks) should be stored somewhere to prevent the same file opening Multiple times (I could add it to the map for css injection and rename it to something more generic like "documentProperties" or "tabProps..."

as far as i can this is already handled, the same file doesn't open twice in testing